test/controllers/experiments_controller_test.rb

```diff
@@ -1,10 +1,45 @@
 require 'test_helper'
 
-class ExperimentsControllerTest < ActionController::TestCase
+
+class ExperimeechntsControllerTest < ActionController::TestCase
+  tests ExperimentsController
```

Comment:
Rzeczywiście... mogę przysiąść że wczoraj jak zacząłem to pisać była dobra nazwa i nie działało, ale nie mogę tego teraz odtworzyć i działa jak jest dobra nazwa więc nie wiem co ja wczoraj z tym zrobiłem.

Przechodzą: ExperimentsController.any_instance.stubs(:authenticate)